app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/MappingWizardDatastoresStep.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form';
+import { noop, bindMethods } from 'patternfly-react';
+import SourceClusterSelect from './components/SourceClusterSelect/SourceClusterSelect';
+import DatastoresStepForm from './components/DatastoresStepForm';
+
+class MappingWizardDatastoresStep extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedCluster: undefined,
+      selectedClusterMapping: null
+    };
+
+    bindMethods(this, ['selectSourceCluster', 'resetState']);
+  }
+
+  componentDidMount() {}
```

Comment:
Does eslint give you a warning here? It should I think, we should remove empty methods.